### DISCUSSION TITLE
Welcome to Bulk-Crap-Uninstaller Discussions!

### DISCUSSION THREAD

**Discussion by Klocman** [Klocman/Bulk-Crap-Uninstaller#287] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by lord0gnome** [comment]
Hey Guys, I just thought someone should congradulate Klocman on a fantastic freaking program. I truly love it and will spread the word of the BCU far and wide. ;)

**Reply by WatchTheory** [reply]
I love using  Bulk- Crap Uninstaller, it came to a point that Windows built-in uninstaller does nothing for me...BCU has everything I need and does the job.. A Congratulation Klocman on beyond fantastic program.

**Comment by yorickkooij** [comment]
Hey Klocman, first of all, BCU has been a great help in bulk removing old junk from my pc and laptop! I'm only running into one issue... Is there a way to automatically confirm on the warning dialog boxes? If I leave my PC during quiet uninstalling currently a lot of my queue times out because I wasn't there to click them...
Picture attached for illustration.
![bcu](https://user-images.githubusercontent.com/4697074/119222714-00054a80-baf6-11eb-99d9-d94d65ef029a.png)

**Reply by Klocman** [reply]
Looks like a nasty "silent" uninstaller that still asks for input. Make sure all of the quiet uninstallation settings are turned on, if that doesn't help then it'll need to be explicitly handled by BCU.

**Comment by ghost** [comment]
It would be nice to have a feature to monitor software installation.

**Reply by GlassGruber** [reply]
I was about to ask if such a feature or similar is present.

**Reply by Klocman** [reply]
#258

**Comment by themrblueguy** [comment]
Hello,
I am new to this website, and I joined because I would like to not only thank you for making this software, which is absolutely the best one I've come across! It is very useful and even saved me from buying Revo Uninstaller! This is perfect!

I also want to mention that since I'm new to this website I don't quite understand how the contribution system works. 
I am a professional Swedish translator with over 6 years of experience, and I would like to translate this into Swedish for the Scandinavian users. As a thank you for making this, and for free at that!
However, I'm not sure how I will get a hold of the English language pack so I can start translating it. Perhaps anyone could assist me in this matter. Thank you!

**Comment by vbelynets** [comment]
Hello!
I would like to ask you to add Ukrainian localization to the program. I can translate this program into Ukrainian.

**Reply by Klocman** [reply]
Hello, that would be great!
Grab the localization pack from [here](https://github.com/Klocman/Bulk-Crap-Uninstaller/releases/tag/v5.6) and check the included readme. You can either send me the translated files or make a PR with the translation if you know how to use git.

**Comment by wayne2056** [comment]
I love the program.  Just sent a donation.   All the best to you!
Wayne

**Comment by Dragon3166** [comment]
i like program is to much ask stop program have pop up window when i usie it therse new version out do you want to update it got very annoying screw it stop see this window on program start update i disbale updates notifcations but that start up winodw still pops up you dont know how irraitating it ispeople should update there choice not be annoyed frustrated see it every start up uninstall something like program besides that annyoance

**Reply by Klocman** [reply]
You can disable pretty much all pop up windows by clicking on a "Do not show this again" checkbox (or similar) in that pop up window. Alternatively you can disable them in Settings > Interface tab > Message boxes. Update notification setting is in the Misc tab.

**Comment by BlkMamba7** [comment]
This uninstaller application is awesome and works really well. However, the interface seems to break when entering menus and settings.
I am using a custom Windows theme in my Win11 install, but this is the first time I've run into issues like this (no other application/ settings and such that use the legacy interface has broken in this manner. Thought I'd share so that this can be potentially addressed with a future update maybe, and I'm mostly assuming that removing my custom theme may fix this issue.

![image_2024-06-02_23-02-15](https://github.com/Klocman/Bulk-Crap-Uninstaller/assets/114163814/fe9e46c1-6b6f-4867-a9fa-eb72e07e767d)
![doc_2024-06-02_23-01-30](https://github.com/Klocman/Bulk-Crap-Uninstaller/assets/114163814/bd531a4b-55e8-41f8-9208-4a4954cf5d44)
![doc_2024-06-02_23-01-21](https://github.com/Klocman/Bulk-Crap-Uninstaller/assets/114163814/0bd22dea-c1b6-4542-b2b0-6f6cb10a0304)

**Reply by m0h4x** [reply]
Hello world,
I've got a little question. I use sccm system center a lot for silent installations. And I also sometimes use bcu for uninstalling several programs when I test the silent deploys or some programs.
Now I was wondering, does bcu generate any log files where I could like take a look at the silent uninstall commands the bcu generates? Cause it would be awesome if I could reuse some of those silent commands to like maybe uninstall some stuff silently with the sccm system center.
Like if I want to install some new version of some package but there's a m'n old version already installed.
So I could first silent remove the old version and then silent install the new version.
Is that something bcu does or I can generate with some parameter? 
Also does it support command line parameters where I could specify like a few programs I want to uninstall silently?
Like bleachbit for example where I can specify for which browsers I want the cache to be cleaned up silently with a parameter.
Thank you.

**Reply by Klocman** [reply]
@m0h4x Select applications you want the commands for on the list and use File > Export script. You can uninstall applications and remove leftovers automatically with bcu-console, see the readme.

**Reply by Klocman** [reply]
@BlkMamba7 First time I've ever seen something like this, so I'm afraid you're on your own here. It must be something about the custom theme messing up the DPI-aware drawing process. Try turning off DPI aware scaling in settings > interface.

**Comment by ghost** [comment]
hi team!

**Reply by Klocman** [reply]
Hi.

**Comment by DavidQ93** [comment]
Hi Klocman!
BCUninstaller works like charm. One of the best progams on my PC. But there is one thing i want to ask. 
BCUninstaller can easily scan files on my C drive, D drive and E drive. And a week ago, i just installed a new drive on my PC and named it F. But i don't know why BC does not scan files on my F drive.
Can you please help me out with this problem? Thank you and sorry for my English.

**Reply by DavidQ93** [reply]
It's been two months. Pls help me, team @Klocman

**Reply by Klocman** [reply]
Maybe it's seen as a removable drive. Go to settings > folders and either put the path in or turn on scanning removables if it is one.

**Comment by Greenlandismine** [comment]
Hello and congratulations on your work. I started using BCU because my usual uninstaller didn't show the internal uninstaller for the program I wanted to remove, so I was afraid it didn't have full control over the uninstallation and wouldn't do a good job.
I've been very happy with it, even though BCU presented me with a thousand windows and a thousand options to get it to work. 
I know how to use a PC, I assemble computers, and I do maintenance for friends and family, what I might suggest is to make a much more basic GUI for common users and then leave the current one for those who need more advanced features.
I had no problems when BCU showed me what I wanted to uninstall on two lines, one green and one blue. However, I think it's confusing and unnecessary, since if I'm on BCU, I want to uninstall EVERYTHING of that program without multiple choices and without anything remaining on my PC.
There's no doubt that I most likely didn't understand everything about using your program, but the fact that I don't speak English certainly didn't help. True, there's a translation, but perhaps there's some explanation to read, and I don't have time.
Best regards and good luck to all of you!!!

**Comment by guidu69** [comment]
Very powerful uninstaller well done

**Comment by werpwerpiop** [comment]
Hi! Firstly, great program. It is my goto for uninstalling on windows. Thanks for your work and for making it available :)
Would it be possible (or is it possible) to exclude drives and folders from the scanning? Some kind of selection list where the user could select which drives to scan?
Thanks!

**Comment by Klocman** [comment]
Closing this discussion since it grew way too big. Feel free to start new discussions!
